protocol/message.go

```diff
@@ -208,3 +208,23 @@ func NewMonitoringProof(ap []*m.AuthenticationPath,
 		},
 	}, ReqSuccess
 }
+
+// GetKey returns the key extracted from
+// a _verified_ CONIKS server's response.
+//
+// If the response contains a single authentication path,
+// the key is obtained from that authentication path.
+//
+// If the response contains a range of authentication paths,
+// the key is obtained from the authentication path corresponding
+// with the latest signed tree roots.
+func (msg *Response) GetKey() []byte {
+	switch df := msg.DirectoryResponse.(type) {
+	case DirectoryProof:
+		return df.AP.Leaf.Value
+	case DirectoryProofs:
+		return df.AP[len(df.AP)-1].Leaf.Value
```

Comment:
Oops, I will fix the doc. 

This would be used in #93, but I should add a test for it, too.